### PR TITLE
boards: add new board Electronut-labs Blip

### PR DESCRIPTION
**Pull request by ntavish**
Added config for https://github.com/electronut/ElectronutLabs-blip/.
Depends on https://github.com/zephyrproject-rtos/zephyr/pull/11837 being merged for the debugger.

### CODE REVIEW

File: boards/arm/nrf52840_blip/doc/nrf52840_blip.rst [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+they may come up as ``/dev/ttyACM0`` and ``/dev/ttyACM1``. The first 
+one of these (``/dev/ttyACM0`` here) is the debugger port, to which 
+GDB can directly connect, without requiring a GDB server with 
+``target external /dev/ttyACM0``, and the second one is acting as a
+serial port, connected to the SoC.
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
```
Comment (carlescufi) [inline boards/arm/nrf52840_blip/doc/nrf52840_blip.rst]: I don't think this is correct, since your board doesn't use J-Link.

**Review by carlescufi — CHANGES_REQUESTED** [review]

**Review by carlescufi — APPROVED** [review]
Thanks!

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by ntavish** [comment]
@dbkinder pushed requested changes.

**Review by dbkinder — APPROVED** [review]
+1 for doc changes

**Comment by ntavish** [comment]
Don't think checkpatch test will pass. The shippable failure is due to `blackmagicprobe.board.cmake` file being not found, since that PR is on hold, because west runners are to be add in the west repo for the time being.

**Comment by ntavish** [comment]
recheck

**Pull request merged**